backend/src/main/resources/reference.conf

```diff
@@ -3,14 +3,16 @@ bootzooka {
 
   db {
     h2 {
+      queueSize = -1
       dataSourceClass = "org.h2.jdbcx.JdbcDataSource"
       properties = {
         url = "jdbc:h2:file:./data/bootzooka"
       }
     }
     postgres {
+      queueSize = -1
       dataSourceClass = "org.postgresql.ds.PGSimpleDataSource"
-      maxConnections = 16
+      maxConnections = 20
```

Comment:
@adamw this was meant to match the number of threads found here:
https://github.com/slick/slick/blob/master/slick/src/main/scala/slick/jdbc/JdbcBackend.scala#L272
I wasn't really concerned with other connections just that these two match.
This way each thread can have it's on connection and not have to wait.